YAFCmodel.Tests/YAFCmodel.Tests.csproj

```diff
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp6.0</TargetFramework>
+    <PlatformTarget>x64</PlatformTarget>

```

Comment:
why? I believe this will prevent it from running on OSX with arm64 architecture

same for the CommandLineToolExample

Reply:
The main YAFC project has been `x64` this whole time, yet I haven't heard of OSX users reporting it not working. Or did you mean something else, and the case of Tests and CLI projects is different?

Reply:
I ran into issue literally last week in my daytime job when upgrading from 8.0.1 to 8.0.2 with some old projects that had PlatformTarget defined as x64 since the beginning. But if the rest of the projects have PlatformTarget x64 it's probably best to keep it all consistent.